docs/docs/administration/customize.md

```
@@ -1186,7 +1188,7 @@ mediasoup:
 
 #### Apply custom settings for TURN server
 
-If always want a specific TURN server configuration, the following to [apply-config.sh](#preserving-customizations-using-apply-confsh) and modify `aaa.bbb.ccc.ddd` and `secret` with your values.
+If you always want a specific TURN server configuration, add the following to [apply-config.sh](#preserving-customizations-using-apply-confsh) and modify `aaa.bbb.ccc.ddd` and `secret` with your values.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)